filters/local.properties

```diff
@@ -91,3 +91,5 @@ environment.build.ldap.url=
 environment.build.ldap.baseDn=
 environment.build.ldap.userName=
 environment.build.ldap.password=
+
+secureSessionCookie=false
```

Comment:
Good point. Also should add a comment to point out the other related property in portal.properties